app/templates/about/legal.hbs

```diff
@@ -1,41 +1,39 @@
-<h1 id="legal">Legal</h1>
+<div class="bg-muted">
```

Comment:
I don't think we should need this wrapping div

Reply:
It was necessary because of the `bg-muted`, without it the default white background shows through, but I think I now understand that those `bg-*` colors are for calling out content and not for the default page. I've removed it. Thanks.